src/VisualStudio/TestUtilities/Window/InteractiveWindow.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using Roslyn.VisualStudio.Test.Utilities.Remoting;
+
+namespace Roslyn.VisualStudio.Test.Utilities
+{
+    public class InteractiveWindow
+    {
+        private const string DteCSharpViewCommand = "View.C#Interactive";
+        private const string DteCSharpWindowTitle = "C# Interactive";
```

Comment:
Resource strings might be flaky as well, since you might have VS in a different language than the host. There should be a GUID registered that we can use instead.